app/Helpers/Geocoder.php

```diff
@@ -23,4 +23,14 @@ public function geocode($location)
             'longitude' => $longitude,
         ];
     }
+
+    public function reverseGeocode($lat, $lng)
+    {
+        $json = file_get_contents("https://maps.googleapis.com/maps/api/geocode/json?latlng=$lat,$lng&key=AIzaSyDb1_XdeHbwLg-5Rr3EOHgutZfqaRp8THE");
```

Comment:
This seems like a good opportunity to change this to get the API key from config, rather than it being exposed in the source!  (Here and in the existing geocode method)